__tests__/api/api.test.js

```diff
@@ -4,32 +4,57 @@ const request = require("supertest");
 const connection = require("../../db/connection");
 const testData = require("../../db/data/test-data");
 
-
 afterAll(() => {
-    return connection.end();
+  return connection.end();
 });
 
 beforeEach(() => {
-    return seed(testData);
+  return seed(testData);
 });
 
-describe('GET /api/topics', () => {
-    test('200: responds with a status of 200', () => {
-      return request(app).get('/api/topics').expect(200);
-    });
+describe("GET /api/topics", () => {
+  test("200: responds with a status of 200", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
 
-    test('200: responds with an array of topics on the body', () => {
-      return request(app)
-        .get('/api/topics')
-        .expect(200)
-        .then(({body}) => {
-          const {topics} = body
-          expect(topics).toBeInstanceOf(Array);
-          expect(topics).toHaveLength(3);
-          topics.forEach((topic) => {
-            expect(topic).toHaveProperty('description', expect.any(String));
-            expect(topic).toHaveProperty('slug', expect.any(String));
-          });
+  test("200: responds with an array of topics on the body", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toBeInstanceOf(Array);
+        expect(topics).toHaveLength(3);
+        topics.forEach((topic) => {
+          expect(topic).toHaveProperty("description", expect.any(String));
+          expect(topic).toHaveProperty("slug", expect.any(String));
         });
-    });
+      });
+  });
+});
+
+describe("GET /api", () => {
+  test("200: responds with a status of 200 and an array of endpoints on the body", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+        expect(body).toHaveProperty("GET /api", expect.any(Object));
+        expect(body).toHaveProperty("GET /api/topics", expect.any(Object));
+        expect(body).toHaveProperty("GET /api/articles", expect.any(Object));
+      });
```

Comment:
Take a look at the jest docs for _.toMatchObject_, it's a useful matcher for asserting against objects and will save you from writing out `.toHaveProperty` so many times over the course of this week 👍